src/openMVG/matching_image_collection/Cascade_Hashing_Matcher_Regions_AllInMemory.cpp

```diff
@@ -144,7 +144,7 @@ void Match
 #endif
     for (int j = 0; j < (int)indexToCompare.size(); ++j)
     {
-      const size_t J = indexToCompare[j];
+      size_t J = indexToCompare[j];
```

Comment:
Strage that J must be non const, since it is just used as a look-up index later on